### ISSUE TITLE
Where is alpha value ? 

### ISSUE THREAD

**Issue by mafar** [narsenico/a-color-picker#19]
Hi,
I am chaning alpha , style is applied to body but why there is no field to show alpha value like there is to show RGB ??

For my use case , it is important to show Alpha value in box RGBA or in HEX string

![image](https://user-images.githubusercontent.com/4101480/65328589-84f2d080-db6b-11e9-88a7-cf80359e0340.png)

**Comment by mafar** [comment]
Is this merged ? Thankx

**Comment by cmann1** [comment]
Yes please, an rgba input, and showing the alpha in the hex would be very useful.
